server/index.js

```diff
@@ -0,0 +1,25 @@
+import express from 'express';
+
+import db from '../sequelize/models/index';
```

Comment:
By default looks for an `index.js` file inside folders. So to be concise you cau use `import db from '../sequelize/models';` with `index` and it will still work